src/VisualStudio/Core/Def/TableDataSource/TaskList/VisualStudioBaseTaskListTable.cs

```diff
@@ -134,10 +134,7 @@ public override IEqualityComparer<TaskListTableItem> GroupingComparer
                 => TaskListTableItem.GroupingComparer.Instance;
 
             public override IEnumerable<TaskListTableItem> Order(IEnumerable<TaskListTableItem> groupedItems)
-            {
-                return groupedItems.OrderBy(d => d.Data.OriginalLine)
-                                   .ThenBy(d => d.Data.OriginalColumn);
-            }
+                => groupedItems.OrderBy(d => d.Data.Span.StartLinePosition);
```

Comment:
StartLinePosition is a LinePosition which is already IComparable.  So we can order by that directly.